Sources/MongoSwift/BSON/BsonValue.swift

```diff
@@ -140,28 +140,36 @@ public struct Binary: BsonValue, Equatable, Codable {
     }
 
     /// Initializes a `Binary` instance from a `Data` object and a `UInt8` subtype.
-    public init(data: Data, subtype: UInt8) {
+    /// Throws an error if the provided data is incompatible with the specified subtype.
+    public init(data: Data, subtype: UInt8) throws {
```

Comment:
Just want to confirm that this is only the code path for manually constructing a Binary (not decoding BSON into a Binary). If so, LGTM.

Reply:
yup, all the other `init`s get routed through here in the end.